hoomd/md/constrain.py

```diff
@@ -75,6 +75,10 @@ class Distance(Constraint):
     ``constraint_group`` and :math:`d_{ij}` is the constraint distance.
     Define any number of constraint groups in the system state.
 
+    Note:
+        Define ``constraint_group`` members and :math:`d_{ij}` in the
+        constraints property of `hoomd.State`.
```

Comment:
Users cannot directly edit a constraints property. They have to go through the snapshot or local snapshot (presumably the snapshot since you cannot set `N_constraints` in a local snapshot). We should mention that in the note here, as there is no direct constraints property of the `State`, and property is a special word idiomatically in Python.